release.py

```diff
@@ -45,12 +45,32 @@ def prepare_for_release(release_version):
     ])
 
 
+def create_data_schema_documentation():
+    try:
+        import pypandoc as converter
+    except ImportError:
+        import pydocverter as converter
+
+    module_folder = os.path.dirname(__file__)
+    docs_folder = os.path.abspath(os.path.join(module_folder, 'docs'))
+    output_file = os.path.abspath(os.path.join(
+        module_folder,
+        'osmaxx', 'conversion', 'converters', 'converter_gis', 'static', 'doc', 'osmaxx_data_schema.html')
+    )
+
+    html = converter.convert(os.path.join(docs_folder, 'osmaxx_data_schema.md'), 'html5')
+    with open(output_file, 'w') as output:
+        output.write(html)
+
+
 def make_release_specific_changes(release_version):
     execute_steps([
         "python ./web_frontend/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
         "python ./osmaxx_conversion_service/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
         ["git", "commit", "-m", 'added makemessages output', 'osmaxx/locale'],
     ])
+    create_data_schema_documentation()
+    execute(["git", "commit", "-m", 'added new schema documentation', 'osmaxx/conversion/converters/converter_gis/static/doc/osmaxx_data_schema.html'])
```

Comment:
Usually, a previous version of the documentation will already have been tracked. Maybe use "synchronized schema documentation HTML" or "updated schema documentation HTML" as commit message?